main.rb

```diff
@@ -6,14 +6,15 @@
 require 'sinatra/reloader'
 require 'json'
 require_relative 'memo'
+require 'webrick'
 
 before do
   @memo = Memo.new
```

Comment:
このオブジェクトはメモのデータを出し入れするデータベース的な役割だと思うので、

```suggestion
  @memo_db = MemoDB.new
```

みたいな名前の方がいいと思います。

もっというと、インスタンス化する必要もないので、

```ruby
MemoDB.read
```

のように全部クラスメソッドにしてしまってもいいと思います

Reply:
DBをつけるだけで読みやすさが全然違ってくるんですね。

初めてクラスメソッドを使ってみました！